.travis.yml

```diff
@@ -24,6 +24,8 @@ install:
     # install ramp-engine
     - cd ramp-engine
     - pip install -e .
+    - cd ../ramp-database
+    - pip install -e .
```

Comment:
I actually don't think the `-e` is needed. Here on travis we simply want to install it no?

Reply:
+1